packages/ui/Scrollable/Scrollbar.js

```diff
@@ -164,6 +172,7 @@ ScrollbarBase.propTypes = /** @lends ui/Scrollable.Scrollbar.prototype */ {
 ScrollbarBase.defaultProps = {
 	corner: false,
 	css: componentCss,
+	minThumbSize: 18,
```

Comment:
Should this `18` be scaled too, or is it scaled elsewhere? What unit does this 18 represent? Pixels?

Reply:
It scaled on line 91. 
> scrollThumbSizeRatio = Math.max(ri.scale(minThumbSize) / trackSize, Math.min(1, thumbSizeRatioBase));

it represent size in pixels